code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -82,7 +82,7 @@
 /obj/machinery/portable_atmospherics/canister/examine(user)
 	. = ..()
 	if(mode)
-		. += "<span class='notice'>This canister is [mode]. A sticker on its side says <b>MAX PRESSURE: [siunit(pressure_limit, "Pa", 0)]</b>.</span>"
+		. += "<span class='notice'>This canister is [mode]. A sticker on its side says <b>MAX PRESSURE: [siunit(pressure_limit * 1000, "Pa", 0)]</b>.</span>"
```

Comment:
Are you sure you want to use the conversion here? Do you think it's clear enough, given that it starts at MPA, is the flavor worth it, etc.

Reply:
Without the *1000 you'll read `"This canister is tier 1. A sticker on its side says MAX PRESSURE: 50 KPa."`
but that's wrong, since the canister can hold 50000 KPa, with the *1000 you'll read `"This canister is tier 1. A sticker on its side says MAX PRESSURE: 50 MPa."` which is more accurate